measures/apps/measuresdata/measuresdata.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# measuresdata.sh
+# Must be executable. Call it with all defaults or arguments for measuresdata
+# It is a test script to enable writing python (or other) versions
+
+echo Calling measuresdata $*
+measuresdata $*
```

Comment:
as an example ./measuresdata.sh 1 "2 3" causes measuresdata to see 3 arguments with `$*` which is wrong, it are two arguments which you get with `$@`